raspberrypi-c-on-device.js

```diff
@@ -80,8 +80,6 @@ function initTasks(gulp, options) {
   });
 
   gulp.task('deploy', 'Deploy and build sample code on the device', function (cb) {
-    // write config file only if any
-    all.writeConfigH();
```

Comment:
I left it for purpose for timebeing, maybe will be still needed by Arduino in some way. Well Arduino will break if we remove it now :-)